snap/info.go

```diff
@@ -409,6 +409,9 @@ func (s *Info) Description() string {
 	return s.OriginalDescription
 }
 
+// GetType returns the type of the snap, specifically also returning TypeSnapd
+// if the snap is the snapd snap, regardless of the actual type of the snapd
```

Comment:
>  regardless of the actual type of the snapd

Wondering about the wording here, doesn't fit exactly with the code.  Perhaps something like:

```
// GetType returns the type of the snap, including additional snap ID check
// for the legacy snapd snap definitions.
```
wdyt?

Reply:
missing . at the end.  Maciej suggestion would also work for me.

Reply:
Pushed this along with a tweak for the other comment.